setup/openchange_provision

```diff
@@ -29,6 +29,24 @@ import samba
 import samba.getopt as options
 import openchange.provision as openchange
 
+def check_incompatible_options(parser, opts):
```

Comment:
For me callbacks will make more sense, you check individually each parameter but if this works, I don't really care